app/user-status-card/user-status-card.component.html

```diff
@@ -0,0 +1,29 @@
+<ion-card *ngIf="user"
+          [navPush]="onClickPage"
```

Comment:
`[navPush]="onClickPage"`

This is an ionic directive that will navigate to the page stored in the variable `onClickPage` when the `ion-card` is clicked. The feed page passes in the `DosePage` as the destination.